plugins/cilium-cni/cmd/cmd.go

```diff
@@ -665,6 +692,13 @@ func (cmd *Cmd) Add(args *skel.CmdArgs) (err error) {
 			if err != nil {
 				return fmt.Errorf("unable to setup interface datapath: %w", err)
 			}
+		case ipamOption.IPAMDelegatedPlugin:
+			if conf.InstallUplinkRoutesForDelegatedIPAM {
+				// TODO: Support IPv6
```

Comment:
What needs to be done to fully support IPV6?

Reply:
@rjdenney The better way is to adjust current ENI logic, add v6 logics along v4 ops. This way Delegated IPAM plugin and ENI IPAM could share same logic. 

Dose azure need v6 for now? Consider the current PR contains too much changes, I will push another one for v6.

Reply:
Yes, for azure we are currently working to get v6 going.

Right now, we are working to get Dualstack VNET Prefix on Secondary NIC. I have done some testing with your changes before you updates yesterday and I could see the routing for the V4 IPs and tested them to make sure they are working but the V6 IPs were not being assigned to the pod or being routed.

Reply:
@ruicao93 V4 does seem to be working well for azure we will just need V6 support. 

And like you mentioned it is working for ip prefix as well and is adding a new rule for each pod. For now this is fine while we test but if you do plan to add either of the two implementations you mentioned let us know.

Also just tagging @tamilmani1989 for visibility.

Reply:
@ruicao93 Also do we have a date that we can expect the V6 changes PR to be out?

Reply:
@rjdenney we can take up for v6

Reply:
@rjdenney @tamilmani1989 I plan to start adapting to IPv6 after this PR is merged. You can take the v6 adaption if urgent.